tests/sqllogictests/suites/base/03_common/03_0041_sort_spill.test

```diff
@@ -0,0 +1,402 @@
+statement ok
```

Comment:
It seems that we don't check the spill or not in this test?

Reply:
We can check the metrics counter.

Reply:
Can we get the metrics by SQL command?

Reply:
Yes, 
`select metric, value from system.metrics where metric like '%<metric-name>%';`